server/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/ConcurrentRebalanceAllocationDecider.java

```diff
@@ -76,4 +76,9 @@ public Decision canRebalance(ShardRouting shardRouting, RoutingAllocation alloca
                 "below threshold [%d] for concurrent rebalances, current rebalance shard count [%d]",
                 clusterConcurrentRebalance, relocatingShards);
     }
+    
+    @Override
+    public Decision canRebalance(RoutingAllocation allocation) {
+        return canRebalance(null, allocation);
```

Comment:
instead of passing `null` here, can you move the implementation to this method and then call call this method from `canRebalance(ShardRouting shardRouting, RoutingAllocation allocation)`, similar as was done for `ClusterRebalanceAllocationDecider`.